### DISCUSSION TITLE
Feature: Ability to render form template

### DISCUSSION THREAD

**Discussion by lemmon** [tobimori/kirby-dreamform#104] [Feature Requests]
The `render()` function always returns error page on GET requests.

https://github.com/tobimori/kirby-dreamform/blob/a843b52bbee1e393a247817d12ba0e9dc3191fe8/classes/Models/FormPage.php#L412

I need to render pages with relevant forms only. I am currently applying workaround by creating a virtual page `viewform/(:any)`. It would be nice that there's an option to allow users render form pages when needed.

**Comment by tobimori** [comment]
will be included in 2.0
